### PR TITLE
Added documentation & fixed misleading information regarding theme-specific modules

### PR DESCRIPTION
**Pull request by sheerlox**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.x
| Description?  | Added documentation & fixed misleading information regarding theme-specific modules <br/><br/> Had a hard time recently finding information about this (nothing in docs apart from misleading information and nothing on the web), had to read the related code to understand how this all worked. <br/> So I figured I'd contribute to the docs :)
| Fixed ticket? | None

### CODE REVIEW

**Comment by sheerlox** [comment]
I'll check that the feature works the same way in 1.8.x and submit a PR on the `8.x` branch in a moment.

File: themes/reference/theme-specific-modules.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Theme-specific modules
+aliases:
+  - /1.7/themes/theme-specific_modules/
+  - /themes/theme-specific_modules/index.html
+---
+
+# Theme-specific modules
+
+When you write a theme, you may need to extend Prestashop features. You can do that by adding modules that will be tied to your theme and shipped with it.
```
Comment (Progi1984) [inline themes/reference/theme-specific-modules.md]:
```suggestion
When you write a theme, you may need to extend PrestaShop features. You can do that by adding modules that will be tied to your theme and shipped with it.
```

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]
I did commit the name feedback from @Progi1984 

Cool contribution tbh 👍 

We also need to add it to the 8.x documentation I think

**Comment by sheerlox** [comment]
Moved to #1165.

**Pull request closed without merging**